packages/wallet-ui/src/components/ui/organism/Header/SendSummaryModal/SendSummaryModal.view.tsx

```diff
@@ -232,7 +232,7 @@ export const SendSummaryModalView = ({ address, amount, chainId, closeModal }: P
           </RightSummary>
         </Summary>
         {!estimatingGas && <TotalAmount>Maximum fees: {gasFeesAmount} ETH</TotalAmount>}
-        {gasFees.includeDeploy && <IncludeDeploy>*Fees include a one-time deployment fee</IncludeDeploy>}
+        {gasFees.includeDeploy && <IncludeDeploy>*Fees include an one-time deployment fee</IncludeDeploy>}
```

Comment:
I think the correct spelling is actually "a" here 
https://www.quora.com/Which-is-correct-a-one-or-an-one#:~:text=It's%20'a%20one%2Dtime%20opportunity,sound%2C%20not%20an%20o%20sound.

Reply:
@Julink-eth oh u r right, nice catch!